src/ui/reducers/timeline.test.ts

```diff
@@ -0,0 +1,218 @@
+import { configureStore, EnhancedStore, Reducer, ThunkDispatch } from "@reduxjs/toolkit";
+import { UIAction } from "ui/actions";
+import { UIState } from "ui/state";
+import { ThunkExtraArgs } from "ui/utils/thunk";
+
+import * as actions from "../actions/timeline";
+
+import appReducer from "./app";
+import timelineReducer, {
+  getCurrentTime,
+  getFocusRegion,
+  getHoverTime,
+  getPlayback,
+} from "./timeline";
+
+type UIStateReducers = {
+  [key in keyof UIState]: Reducer<UIState[key]>;
+};
+
+describe("Redux timeline state", () => {
+  let dispatch = null as unknown as ThunkDispatch<UIState, ThunkExtraArgs, UIAction>;
+  let store = null as unknown as EnhancedStore;
```

Comment:
:hand: Not particularly critical here, but normally I'd actually do a real `configureStore()` here just to get the right types assigned to `store`, since there is a _lot_ of inference going on.  Then `let dispatch = store.dispatch` would also skip all the `ThunkDispatch` stuff

Reply:
Sure. I was having a brain fart moment. Should have just used the `createTestStore` helper util. I'll swap that in.

As for the typing stuff– isn't that still necessary, since the variables are test scope and the store isn't created until the `beforeEach`, no? Happy to change that if you have a more concrete suggestion.